CMSgov/design-system
# [NO_TICKET] Fix flag typing

## Summary
- Include type defs for feature flag
- Rely on generating type defs for tsx files

## How to test
- Run the CMSDS in CRA typescript, try using the feature flag functions and other components to ensure the definitions are working as expected
